libraries/AC_SafeRTL/AC_SafeRTL.cpp

```diff
@@ -0,0 +1,573 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "AC_SafeRTL.h"
+
+extern const AP_HAL::HAL& hal;
+
+const AP_Param::GroupInfo AC_SafeRTL::var_info[] = {
+    // @Param: ACCURACY
```

Comment:
Not sure I agree with renaming from ACCURACY to INTERVAL.  For me "interval" is normally a time thing - and the interval of saving points is actually 3hz at the moment (controlled from the vehicle code actually).  "accuracy" is a better word I think because it's the minimum distance between points and I think it may also be used as a minimum distance during simplification.